### PR TITLE
Reduce conserv record validations

### LINKED ISSUE #359: Adjust validations to accommodate partial cataloging
From reviewer:
> Some books don’t have full records or are uncatalogued. Because of this, the following popped up when trying to enter data for an item with partial cataloging data.  See screen shot below. A solution: N/A was used to fill these fields.  Is this the best going forward?
>
> image

### LINKED ISSUE #381: Reduce validations for Conservation Records
* Verify validations for only:
    title, item_record_no, department, and date_received
* Remove other validations
* Label required form-fields

### PR DESCRIPTION
**Pull request by crowesn**
Resolves #381 
Resolves #359

### CODE REVIEW

File: app/views/conservation_records/_form.html.erb [hunk 1]
```diff
@@ -14,17 +14,17 @@
...
     <div class="form-group">
-      <%= form.label :department %>
+      <%= form.label :department %> <span style="color: #e00122;">*</span>
       <%= form.select(:department, options_from_collection_for_select(@departments, "id", "key", @conservation_record.department), { prompt: 'Select Department' }, {class: 'form-control'}) %>
     </div>
 
   <div class="form-group">
-    <%= form.label :title %>
+    <%= form.label :title %> <span style="color: #e00122;">*</span>
     <%= form.text_field :title, class: 'form-control' %>
```
Comment (hortongn) [inline app/views/conservation_records/_form.html.erb:28]: For required fields, can you also add a `required: true` to the end?  Something like `<%= form.text_field :title, class: 'form-control', required: true %>`

It's considered good practice.  It catches the empty field before the user submits and I think it helps with accessibility by adding some markup that helps people with screen readers detect required fields.

File: app/views/conservation_records/_form.html.erb [hunk 2]
```diff
@@ -14,17 +14,17 @@
...
   <div class="form-block">
     <div class="form-group">
-      <%= form.label :date_received_in_preservation_services %>
+      <%= form.label :date_received_in_preservation_services %> <span style="color: #e00122;">*</span>
       <%= form.date_field :date_received_in_preservation_services, class: 'form-control' %>
     </div>
 
     <div class="form-group">
-      <%= form.label :department %>
+      <%= form.label :department %> <span style="color: #e00122;">*</span>
```
Comment (hortongn) [inline app/views/conservation_records/_form.html.erb:22]: To DRY this up a bit, can you move the color specification to application.scss and just use something like `<span class="required">*</span>` in the views?  That puts the color code in a single place and makes it easier to change down the road.

There are also ways to use CSS pseudo-classes to automatically add an asterisk to required field labels, but that's probably getting more complicated that it needs to be.

**Review by hortongn — CHANGES_REQUESTED** [review]
@crowesn See my inline comments.

File: app/views/conservation_records/_form.html.erb (hunk 1 above)
Comment (hortongn) [reply app/views/conservation_records/_form.html.erb:28]: @crowesn I don't see that you've added `required: true` like I suggested.  Any reason for not adding that to required fields?

Comment (crowesn) [reply app/views/conservation_records/_form.html.erb:28]: apologies, I thought I had read this suggestion but then I didn't see it on a second reading. I will add.

**Review by hortongn — APPROVED** [review]
@crowesn Thanks for the tweaks.  merging

**Pull request merged**